x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -91,8 +83,15 @@ const installPreBuiltTemplates = async (
     // The existing convenience endpoint `indices.putTemplate` only sends to _template,
     // which does not support v2 templates.
     // See src/core/server/elasticsearch/api_types.ts for available endpoints.
-    await callCluster('transport.request', callClusterParams);
+    return await callCluster('transport.request', callClusterParams);
   });
+  try {
+    return await Promise.all(templateInstallPromises);
+  } catch (e) {
+    throw new Boom(`Error installing prebuilt index templates ${e.message}`, {
```

Comment:
Do we want to start to introduce our domain errors and avoid using `Boom` ?

Reply:
I would prefer to start with that in changes that don't go into 7.8 to keep the changes smaller.